crates/subspace-farmer/src/plotter/gpu/cuda.rs

```diff
@@ -93,8 +94,24 @@ impl CudaRecordsEncoder {
         global_mutex: Arc<AsyncMutex<()>>,
     ) -> Result<Self, ThreadPoolBuildError> {
         let id = cuda_device.id();
+        let thread_name = move |thread_index| format!("cuda-{id}.{thread_index}");
+        // TODO: remove this panic handler when rayon logs panic_info
```

Comment:
Please add a link to upstream issue in this TODO or else it is not clear what and when is supposed to happen for this to be resolved.

Reply:
Sure, happy to open one tomorrow, or if you get to it first, I'll add that link

Reply:
```suggestion
        // TODO: remove this panic handler when rayon logs panic_info: https://github.com/rayon-rs/rayon/issues/1208
```